grunttasks/l10n-htmllint.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.registerTask('l10n-htmllint', 'Lint l10n files', function () {
+    // grunt addition for htmllint
+  });
```

Comment:
Not sure what this task is yet.

I was running it with this, but it's EXTREMELY heavy since it does a full build. Not sure what the minimum subset of tasks is to clean `dist` and then generate the `/dist/i18n/` directory with .JSON files.

```js
  grunt.registerTask('l10n-htmllint', 'Lint l10n files', [
    'build',
    'l10n-json-to-html',
    'htmllint:l10n'
  ]);
```

Where my local grunttasks/htmllint.js file is roughly this (where I copied your rules from your **.htmllintrc**, but didn't convert double to single quotes yet):
```js
  grunt.config('htmllint', {

    options: {
      "attr-no-dup": true,
      "attr-no-unsafe-char": true,
      "attr-quote-style": "quoted",
      "doctype-first": false,
      "id-class-style": false,
      "indent-style": "spaces",
      "indent-width": 0,
      "tag-name-lowercase": true,
      "tag-name-match": true
    },

    l10n: [
      '<%= yeoman.tmp %>/i18n/*/*.html'
    ],
```